### PR TITLE
Coverage increased in utils

### LINKED ISSUE #667: Increase coverage for v1 utils

### PR DESCRIPTION
**Pull request by shekhar2807**
# Description

-Tests added for utils which were not covered in V1.
-some additional tests in CMD.

Fixes #667

### CODE REVIEW

**Comment by ashish10677** [comment]
@shekhar2807 please write tests for merkle.go

**Comment by shekhar2807** [comment]
> @shekhar2807 please write tests for merkle.go

Done @ashish10677

File: utils/merkle_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		},
+		{
+			name: "Test 2: When it contains multiple values",
+			args: args{
+				values: []*big.Int{big.NewInt(1), big.NewInt(1), big.NewInt(1)},
+			},
+			want: [][][]byte{{{52, 34, 71, 138, 66, 123, 171, 13, 20, 101, 234, 111, 34, 80, 119, 150, 191, 7, 189, 164, 50, 31, 68, 19, 51, 45, 101, 139, 128, 81, 172, 181}}, {{182, 253, 152, 238, 183, 224, 143, 197, 33, 241, 21, 17, 40, 154, 254, 77, 142, 135, 63, 215, 163, 251, 118, 171, 117, 127, 164, 124, 35, 245, 150, 233}, {177, 14, 45, 82, 118, 18, 7, 59, 38, 238, 205, 253, 113, 126, 106, 50, 12, 244, 75, 74, 250, 194, 176, 115, 45, 159, 203, 226, 183, 250, 12, 246}}, {{177, 14, 45, 82, 118, 18, 7, 59, 38, 238, 205, 253, 113, 126, 106, 50, 12, 244, 75, 74, 250, 194, 176, 115, 45, 159, 203, 226, 183, 250, 12, 246}, {177, 14, 45, 82, 118, 18, 7, 59, 38, 238, 205, 253, 113, 126, 106, 50, 12, 244, 75, 74, 250, 194, 176, 115, 45, 159, 203, 226, 183, 250, 12, 246}, {177, 14, 45, 82, 118, 18, 7, 59, 38, 238, 205, 253, 113, 126, 106, 50, 12, 244, 75, 74, 250, 194, 176, 115, 45, 159, 203, 226, 183, 250, 12, 246}}},
```
Comment (ashish10677) [inline utils/merkle_test.go:30]: Please write a few more test cases with varied values.

Comment (ashish10677) [reply utils/merkle_test.go:30]: Add tests if values is even, odd or 0.

File: utils/merkle_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		args args
+		want [32]byte
+	}{
+		{
+			name: "When GetMerkleRoot() executes successfully",
+			args: args{
+				tree: [][][]byte{{{2}, {1}, {3}}},
+			},
+			want: [32]byte{2},
+		},
```
Comment (ashish10677) [inline utils/merkle_test.go:58]: Write tests if tree is nil or has no elements

File: utils/merkle_test.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+		{
+			name: "Test 2: When GetProofPath() executes successfully and assetId is odd",
+			args: args{
+				tree:    [][][]byte{{{1, 10}, {2, 12}, {3, 13}}, {{4, 14}, {5, 15}, {6, 16}}, {{7, 17}, {8, 18}, {9, 19}}},
+				assetId: 1,
+			},
+			want: [][32]byte{{7, 17}, {5, 15}},
+		},
+	}
```
Comment (ashish10677) [inline utils/merkle_test.go:96]: What happens if tree is nil

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: utils/merkle_test.go (hunk 1 above)
Comment (shekhar2807) [reply utils/merkle_test.go:30]: Done @ashish10677

File: utils/merkle_test.go (hunk 2 above)
Comment (shekhar2807) [reply utils/merkle_test.go:79]: Done @ashish10677

File: utils/merkle_test.go (hunk 3 above)
Comment (shekhar2807) [reply utils/merkle_test.go:124]: The test does not run for this particular case . @ashish10677

Comment (ashish10677) [reply utils/merkle_test.go:124]: Can you write that test when tree is nil.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: utils/merkle_test.go (hunk 3 above)
Comment (shekhar2807) [reply utils/merkle_test.go:132]: Done  @ashish10677

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**